pkg/arvo/mar/group/update.hoon

```diff
@@ -8,9 +8,4 @@
   %+  frond:enjs:format  'groupUpdate'
   (update:enjs upd)
   --
-++  grab
-  |%
-  ++  noun  update
```

Comment:
?

Reply:
artifact of splitting up update and action - with new idea, see below, I'd reinstate this